doc/schemas/lightning-listpeerchannels.json

```diff
@@ -546,6 +546,13 @@
                 "Max amount accept in a single payment."
               ]
             },
+            "max_total_htlc_out_msat": {
+              "type": "msat",
+	      "added": "v24.05",
+              "description": [
+                "Max total amount that can be sent, independently on how much is on our side."
+              ]
+            },
```

Comment:
```
    |
    = note: `JsonRpcCodec` has a derived impl for the trait `Default`, but this is intentionally ignored during dead code analysis
    = note: `#[warn(dead_code)]` on by default

--- doc/schemas/lightning-listpeerchannels.json	2024-10-30 11:09:30.533230262 +0000
+++ doc/schemas/lightning-listpeerchannels.json.fmt	2024-10-30 11:20:59.933059209 +0000
@@ -548,7 +548,7 @@
             },
             "max_total_htlc_out_msat": {
               "type": "msat",
-	      "added": "v24.05",
+              "added": "v24.05",
               "description": [
                 "Max total amount that can be sent, independently on how much is on our side."
               ]
make: *** [doc/Makefile:176: check-fmt-schema/doc/schemas/lightning-listpeerchannels.json] Error 1
```

probably from CI

I am not familiar anymore with the docs procedure, but there is a way to fix it

Reply:
Ha. Yes. I had this PR since May I have to change v24.05 to v24.11.
Thanks.